website/docs/r/log_search.html.markdown

```diff
@@ -29,6 +29,19 @@ resource "sumologic_log_search" "example_log_search" {
             }
         }
     }
+    
+    query_parameter {
```

Comment:
Have you tested this?

For this example to make more sense the `query_string` should use the query parameters defined here.

Reply:
This hasn't been tested, because as-is this does not work. If the `query_parameter`s don't match up with the `query_string` then you'll get this error, which I found when testing this locally:
```
│ Error: {"id":"<removed>","errors":[{"code":"ss:parameter_mismatch","message":"Provided log search is invalid.","detail":"Error creating scheduled search parameters","meta":{"validationProblems":{}}}]}
│ 
│   with sumologic_log_search.example_log_search,
│   on personal.tf line 3, in resource "sumologic_log_search" "example_log_search":
│    3: resource "sumologic_log_search" "example_log_search" {
```

Reply:
modified example to include valid query paramters which are being used in search